build-plugin/debian/rules

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/make -f
+%:
+	dh $@
+override_dh_install:
+	dh_install signalfx_metadata.py  opt/signalfx-collectd-plugin
```

Comment:
I think we are going to end up with a whole directory of files.. lets figure out how to deal with that.